.ibm/pipelines/value_files/values_showcase-rbac.yaml

```diff
@@ -151,9 +151,19 @@ upstream:
         cors:
           origin: 'https://{{- include "janus-idp.hostname" . }}'
         database:
-          connection:
-            password: ${POSTGRESQL_ADMIN_PASSWORD}
-            user: postgres
+          connection:  # configure Backstage DB connection parameters
+            host: ${POSTGRES_HOST}
+            port: ${POSTGRES_PORT}
+            user: ${POSTGRES_USER}
+            password: ${POSTGRES_PASSWORD}
+            ssl:
+              rejectUnauthorized: false,
```

Comment:
What if you set rejectUnauthorized: true ?

Reply:
https://github.com/backstage/charts/issues/80#issuecomment-1501776357

`rejectUnauthorized: true` also works. but with false it will tell not to fail if the connection isn't verified.

I have update the `rejectUnauthorized` to true.